test/async-hooks/test-run-in-async-id-scope.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const async_hooks = require('async_hooks');
+
+const asyncId = async_hooks.newUid();
+
+assert.notStrictEqual(async_hooks.currentId(), asyncId);
+
+async_hooks.runInAsyncIdScope(asyncId, common.mustCall(() => {
+  assert.strictEqual(async_hooks.currentId(), asyncId);
+}));
```

Comment:
I think this test can go into `parallel` (it doesn’t use any of the async hook test helpers here)